bin/servicerouter.py

```diff
@@ -811,6 +811,11 @@ def wsgi_app(env, start_response):
     httpd.serve_forever()
 
 
+def clear_callbacks(marathon, callback_url):
```

Comment:
nit: I don't see a lot of value in having this be a function rather than be inline. That said, works fine either way.